src/Autofac/Autofac.csproj

```diff
@@ -48,19 +48,21 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.1">
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="6.0.0" />
```

Comment:
This `Microsoft.Bcl.AsyncInterfaces` package was originally inside a conditional include so it only applied for `netstandard2.0`. It needs to stay conditional. I don't think it needs to even be updated if it's in the conditional, and that'd be fine. We don't want to force folks to take package updates.